packages/web/src/components/Home.tsx

```diff
@@ -1,15 +1,20 @@
 import AlgoliaBadge from "./AlgoliaBadge";
 import Container from "@material-ui/core/Container";
 import { Grid } from "@material-ui/core";
+import { Helmet } from "react-helmet";
 import HomeButtons from "./HomeButtons";
 import React from "react";
 import SearchForm from "./SearchForm";
 import { useTranslation } from "react-i18next";
 
 const Home = () => {
-  const { t } = useTranslation("footer");
+  const { t } = useTranslation(["footer", "home"]);
```

Comment:
Another options could be removing the specific name space here and just applying it within the t function, e.g.:
```js
const { t } = useTranslation();

// ...other stuff in between

<Helmet>
   <title>{t("home.title")}</title>
   <meta name="description" content={t("home.description")} />
</Helmet>
```
NBD either way but thought this may be a bit easier to read.